test/bitswap.spec.js

```diff
@@ -50,13 +68,39 @@ describe('.bitswap', () => {
       })
     })
 
+    it('.stat (modular)', (done) => {
```

Comment:
It would be nice to find a way to not duplicate the tests, but rather reuse them and just pass in the two different versions